debian/control

```diff
@@ -4,53 +4,24 @@ Section: misc
 Priority: optional
 Standards-Version: 3.9.2
 Build-Depends:
- clang-3.8,
- debhelper (>= 9),
- libgflags-dev,
- libgoogle-glog-dev,
- libudev-dev,
- libxml2-dev,
- libjsoncpp-dev,
  cdbs,
  debhelper,
- dh-buildinfo,
- config-package-dev (>= 4.5~),
- libnl-3-dev,
- libnl-genl-3-dev
+ config-package-dev (>= 4.5~)
 
 Package: cuttlefish-common
 Architecture: any
 Depends:
- acl,
- binutils,
+ binfmt-support [!amd64],
  bridge-utils,
- device-tree-compiler,
- dnsmasq-base,
- hostapd,
- simg2img [amd64],
- simg2img:amd64 [!amd64],
- iptables,
  libarchive-tools | bsdtar,
+ libc6:amd64 [!amd64],
  libdrm2,
- libc6,
- libgoogle-glog0v5 | libgoogle-glog0,
- libnl-genl-3-200,
- libstdc++6,
- libudev1,
+ libfdt1,
  libwayland-client0,
- libwayland-server0,
```

Comment:
I thought this was necessary for crosvm?

https://github.com/google/android-cuttlefish/commit/8706386b1b053834a929670be58f5a9cb6a0627e

Reply:
Why would the Wayland server be needed on the rock pi?  Either way, I was able to run under crosvm without it.

Reply:
You're right, I'm also able to launch without it on GCE.